asset-transfer-basic/chaincode-external-nano/README.md

```diff
@@ -0,0 +1,78 @@
+# Asset-Transfer-Basic as an external service - Nano
+
+This sample complements the [Test Network Nano sample](../../test-network-nano-bash) demonstrating how you can run the chaincode as an external service without using any container at all.
+
+## Setting up the external builder and launcher
+
+Fabric's external builders and launchers is an advanced feature that typically requires custom packaging of the peer image so that it contains all the tools your builder and launcher require. This sample uses very simple (and crude) shell scripts that can be run directly by the default Fabric peer.
```

Comment:
The latest `core.yaml` and the latest 2.4 binaries have a preconfigured external builder for 'chaincode-as-a-service' these are go applications - so don't need to use 'the crude shell scripts'

Though ironically the scripts used are mentioned in the docs :-)

Reply:
Yup, I'm well aware of the ccaas builder. I actually wanted to use it but that's not a binary that we make available independently and that people will have on their machine so I decided against using it.
If we do want to use it we will have to instruct people on how to build it first.

Reply:
The ccaas builder is in the same tgz file as the other binaries... so if the install script is used to pull those - you can the ccaas for *free*

Reply:
I just downloaded the binaries for 2.4.2 and it's not there.

Reply:
Well, even on jfrog it's only included in the linux distro. 👎

Reply:
So, I switched to using ccaasbuilder adding instructions to build it for the poor souls that won't find it from installing the binaries. We can eventually remove those instructions.